supertokens_python/recipe/thirdpartyemailpassword/asyncio/__init__.py

```diff
@@ -62,6 +63,29 @@ async def get_user_by_third_party_info(
     )
 
 
+async def thirdparty_sign_in_up(
```

Comment:
this function is not to be exposed in all the third party related recipes. It's there in the node PR, but it is removed later on - im not sure if it's removed in a PR that is listed.. so remove it here

Reply:
Done